### PR TITLE
chore: move content and features to templates

### PR DESCRIPTION
**Pull request by redreceipt**
this will allow us to migrate a couple of the pkgs that were cross-dependent from `ui-connected`

### CODE REVIEW

**Review by calebpanza — CHANGES_REQUESTED** [review]
CleanShot 2022-11-01 at 10 07 30

**Comment by redreceipt** [comment]
@calebpanza I just pulled this branch and booted again no problem, how did you get to see this error? did you nav somewhere in the app?

Screenshot 2022-11-01 at 11 48 50 AM

**Comment by redreceipt** [comment]
> @redreceipt all I ended up doing was `yarn && yarn start`. I'll try it again

did you yarn in the root as well? symlinked pkgs are now different as well

**Comment by calebpanza** [comment]
@redreceipt looks like there was a missing import on the `createTab`. I bet that the `ui-connected` package was cached on your local machine or something

**Comment by redreceipt** [comment]
> @redreceipt looks like there was a missing import on the `createTab`. I bet that the `ui-connected` package was cached on your local machine or something

boom, I bet you're right. Thank you!

**Review by calebpanza — APPROVED** [review]

File: templates/mobile/__mocks__/@apollosproject/ui-kit.js [hunk 1]
```diff
@@ -44,5 +47,20 @@ export const ErrorText = 'ErrorText';
...
+export const ActionList = 'ActionList';
+export const ActionCard = 'ActionCard';
+export const AddCommentInput = 'AddCommentInput';
+export const Cell = 'Cell';
+export const CellIcon = 'CellIcon';
+export const CellText = 'CellText';
+export const CellContent = 'CellContent';
+export const ActionBar = 'ActionBar';
+export const ActionBarItem = 'ActionBarItem';
+export const CenteredView = 'CenteredView';
```
Comment (redreceipt) [inline templates/mobile/__mocks__/@apollosproject/ui-kit.js:66]:
```suggestion
export const CenteredView = 'CenteredView';

```

**Review by vinnyjth — APPROVED** [review]

**Comment by vinnyjth** [comment]
Love it! thanks man

**Comment by redreceipt** [comment]
Not sure why the test is failing here. Might involve some tricky GitHub actions debug.

**Comment by redreceipt** [comment]
test only fails on linux, jest is failing with an exit code of 1 but passing all tests...
Screenshot 2022-11-02 at 9 50 43 AM

**Comment by redreceipt** [comment]
it's badgelist and something with reanimated

Screenshot 2022-11-02 at 9 50 43 AM

**Comment by redreceipt** [comment]
got it. animated stuff needs fake timers

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -73,6 +73,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v3
+      # remove once tests are passing
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
```
Comment (redreceipt) [inline .github/workflows/ci.yml:78]:
```suggestion
```

**Pull request merged**